test.ts

```diff
@@ -40,136 +72,718 @@ describe('does parsing', () => {
 
 describe('supported value types', () => {
   it('boolean without following argument', () => {
-
+    const [options] = parse(
+      ['-stars', '-q'],
+      {
+        'stars': {
+          alias: 's',
+          desc: 'oh I believe in yesterday.',
+          type: 'boolean'
+        },
+        'quiet': {
+          alias: 'q',
+          desc: 'the quiet one.',
+          type: 'boolean'
+        }
+      }
+    );
+    expect(options).toEqual({
+      stars: true,
+      quiet: true,
+    });
   });
 
   it("boolean with prefixed 'no'", () => {
-
+    const [options] = parse(
+      ['--no-stars', '--no-quiet'],
+      {
+        'no-stars': {
+          alias: 'y',
+          desc: 'oh I believe in yesterday.',
+          type: 'boolean'
+        },
+        'no-quiet': {
+          alias: 'q',
+          desc: 'the quiet one.',
+          type: 'boolean'
+        }
+      }
+    );
+    expect(options).toEqual({
+      'no-stars': undefined,
+      'no-quiet': undefined
+    });
   });
 
   it('boolean with rhs true', () => {
-
+    const [options] = parse(
+      ['--stars=true', '--moon=true'],
+      {
+        'stars': {
+          alias: 'y',
+          desc: 'oh I believe in yesterday.',
+          type: 'boolean'
+        },
+        'moon': {
+          alias: 'q',
+          desc: 'the quiet one.',
+          type: 'boolean'
+        }
+      }
+    );
+    expect(options).toEqual({
+      'stars': true,
+      'moon': true
+    });
   });
 
   it('boolean with rhs false', () => {
-
+    const [options] = parse(
+      ['--stars=false', '--moon=false'],
+      {
+        'stars': {
+          alias: 'y',
+          desc: 'oh I believe in yesterday.',
+          type: 'boolean'
+        },
+        'moon': {
+          alias: 'q',
+          desc: 'the quiet one.',
+          type: 'boolean'
+        }
+      }
+    );
+    expect(options).toEqual({
+      'stars': false,
+      'moon': false
+    });
   });
 
+  // TODO: it's ok ?
   it("boolean with any value that not equal to 'false'", () => {
-
+    const [options] = parse(
+      ['--stars=true', '--moon=true'],
```

Comment:
let's use any value other than true and false to test. --stars=whatever --moon=however